VMBackup/main/ScriptRunner.py

```diff
@@ -102,6 +102,8 @@ def get_config(self):
                 self.postScriptNoOfRetries = configData['postScriptNoOfRetries']
             if 'fsFreezeEnabled' in configDataKeys:
                 self.fsFreeze_on = configData['fsFreezeEnabled']
+            if 'ScriptsExecutionPollTimeSeconds' in configDataKeys and int(configData['ScriptsExecutionPollTimeSeconds']) >= 1 and int(configData['ScriptsExecutionPollTimeSeconds']) <=5:
+                self.pollSleepTime = int(configData['ScriptsExecutionPollTimeSeconds'])
```

Comment:
add try-except ValueError to handle non-int case

Reply:
It is already in bigger try-catch.